C++/Language/Algorithm/Sorting/BubbleSort.cpp

```diff
@@ -13,16 +13,15 @@ void swap(int *xp, int *yp)
 // A function to implement bubble sort
 void bubbleSort(int arr[], int n)
 {
-	int i, j;
-	for (i = 0; i < n-1; i++)	
-	
-	// Last i elements are already in place
-	for (j = 0; j < n-i-1; j++)
-		if (arr[j] > arr[j+1])
-			swap(&arr[j], &arr[j+1]);
+	for (int i = 0; i < n - 1; i++)
+
+		// Last i elements are already in place
+		for (int j = 0; j < n - i - 1; j++)
+			if (arr[j] > arr[j + 1])
+				swap(&arr[j], &arr[j + 1]);
 }
 
-/* Function to print an array */
+/*Function to print an array */
 void printArray(int arr[], int size)
 {
 	int i;
```

Comment:
No need to declare outside this variable you can directly use lik
`for (int i = 0; i < n; i++)`
and also add block